### PR TITLE
Fixes:5165

### PR DESCRIPTION
**Pull request by Shrivaibavi**
Fixes: #5165 

Signed-off-by: Shrivaibavi Raghaventhiran <sraghave@redhat.com>

### CODE REVIEW

File: tests/manage/z_cluster/nodes/test_toleration.py [hunk 1]
```diff
@@ -7,7 +7,11 @@
     check_toleration_on_pods,
     wait_for_pods_to_be_running,
 )
-from ocs_ci.ocs.node import taint_nodes, untaint_nodes, get_ocs_nodes
+from ocs_ci.ocs.node import (
+    get_ocs_nodes,
+    taint_nodes,
+    untaint_nodes,
```
Comment (am-agrawa) [inline tests/manage/z_cluster/nodes/test_toleration.py:13]: I think it's not used anywhere

**Comment by am-agrawa** [comment]
Can we run the PR validation please?

File: tests/manage/z_cluster/nodes/test_toleration.py (hunk 1 above)
Comment (keesturam) [reply tests/manage/z_cluster/nodes/test_toleration.py:13]: `untaint_nodes` is done as part of the finalizer.

**Review by keesturam — APPROVED** [review]
LGTM

File: tests/manage/z_cluster/nodes/test_toleration.py [hunk 2]
```diff
@@ -43,7 +47,8 @@ def test_toleration(self):
         """
         # taint nodes if not already tainted
         nodes = get_ocs_nodes()
-        taint_nodes(nodes)
+        for node in nodes:
+            taint_nodes([node.name])
```
[the remark this answers is not in the document]
Comment (prsurve) [reply tests/manage/z_cluster/nodes/test_toleration.py:51]: only ocs node will be tainted

**Review by prsurve — DISMISSED** [review]

**Review by Akarsha-rai — APPROVED** [review]

**Review by prsurve — APPROVED** [review]

**Pull request merged**